core/src/main/scala/tofu/common/Display.scala

```diff
@@ -0,0 +1,72 @@
+package tofu.common
+import cats.{Eval, Show}
+
+/** Configurable and performant conversion to String */
+trait Display[A] extends Show[A] {
```

Comment:
How good is the idea of extending Show?

Reply:
This is the only way to operate with code that requires Show without using nasty imports.